### ISSUE TITLE
Error Line 395

### ISSUE THREAD

**Issue by xZeroMCPE** [SurvivalHive/Minecraft-Pocket-Edition#3]
It says their is a error in line 395 in Main.php

Please fix thanks, :( 

and also how do i set it up

Line 395: $player->teleport($this->getOwner()->getServer()->getLevelByName($arenaname)->getSafeSpawn(new Position($coords[0],$coords[1],$coords[2])));

**Comment by inxomnyaa** [comment]
If you have an error cant you post the crash report or the whole line?

**Comment by xZeroMCPE** [comment]
Edit: Here is it cause the error was in line  395

$player->teleport($this->getOwner()->getServer()->getLevelByName($arenaname)->getSafeSpawn(new Position($coords[0],$coords[1],$coords[2])));

@thebigsmileXD thebigsmileXD

**Comment by indiemini** [comment]
Same problem here.

**Comment by ZhenMC** [comment]
Not working :(

**Comment by inxomnyaa** [comment]
Wheres the file?
I can't find it and i know how to fix.

``` php
$player->teleport($this->getOwner()->getServer()->getLevelByName($arenaname)->getSafeSpawn();
```

**Comment by xZeroMCPE** [comment]
:D i've started my own SG plugin. It's ready: www.github.com/andrethegamer/survivalgame

NVM about the line 395, im over it :+1:

**Comment by inxomnyaa** [comment]
Oh okay, still, i would love this to be fixed ;)

**Comment by StephanToGo** [comment]
i try to fix the survival plugin in the next days/weeks... can someone tell me what @AndreTheGamer mean with "NVM" ? dont understand it...

And @thebigsmileXD do u know  what changed by chest filling or signs? i get errors by tiles...

**Comment by inxomnyaa** [comment]
Nvm = nevermind

Maybe.. Its because tiles are handled b4, i have to check that

**Comment by inxomnyaa** [comment]
Uhm.. Maybe php7?

**Comment by StephanToGo** [comment]
i think i fixed all bugs for 0.14 please try it out
i test it with imagical mine
